modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Questions.es.js

```diff
@@ -370,7 +370,7 @@ export default withRouter(
 		]);
 
 		function buildURL(search, page, pageSize) {
-			let url = (context.historyRouterBasePath || '/#') + '/questions';
+			let url = (context.historyRouterBasePath || '#') + '/questions';
```

Comment:
Looks OK.
One thing we could do to validate the URL (if that's what we want of course) is to use the [`URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL) API